src/composables/Project.js

```diff
@@ -120,17 +122,50 @@ export async function fetchGit(project) {
   return GitEvent.FetchEvent.next();
 }
 
+async function isDirectory(path) {
+  const stat = await new Promise((resolve) => {
+    fs.stat(
+      path,
+      (e, rv) => resolve(rv),
+    );
+  });
+  return stat.isDirectory();
+}
+
+async function readDir(path) {
+  return new Promise((resolve) => {
+    fs.readdir(
+      path,
+      (e, rv) => resolve(rv),
+    );
+  });
+}
+
+async function getFiles(files, projectId, filename) {
```

Comment:
if I understand function, the fonction would be rename into setFiles, because we populated the files array.

Reply:
Done